docs/source/DeveloperGuide/CodingConvention/index.rst

```diff
@@ -27,114 +65,114 @@ are split into the following standard subpackages:
 Some System packages might also have additional
 
   *test*
-    Any unit tests and other testing codes  
-  *Web*   
-    Web portal codes following the same structure as described in 
+    Any unit tests and other testing codes
+  *Web*
+    Web portal codes following the same structure as described in
     :doc:`../AddingNewComponents/DevelopingWebPages/index`.
-     
-Packages are sets of Python modules and eventually compilable source code 
-together with the instructions to use, build and test it. Source code files are 
+
+Packages are sets of Python modules and eventually compilable source code
+together with the instructions to use, build and test it. Source code files are
 maintained in the SVN code repository.
 
-**R1** 
-  Each package has a unique name, that should be written such that each word starts 
-  with an initial capital letter ( "CamelCase" convention ). *Example*: 
+**R1**
+  Each package has a unique name, that should be written such that each word starts
+  with an initial capital letter ( "CamelCase" convention ). *Example*:
   *DataManagementSystem*.
-  
+
 Module Coding Conventions
---------------------------------  
-       
-**R2** 
-  The first line of every file should contain *$HeadURL: $* macro, which SVN 
-  translates into the Author name and the Date of the most recent commit operation. 
-  This macro is typically placed in the first comment line of the module. 
+--------------------------------
+
+**R2**
+  The first line of every file should contain *$HeadURL: $* macro, which SVN
+  translates into the Author name and the Date of the most recent commit operation.
+  This macro is typically placed in the first comment line of the module.
   Ignored with git.
-  
+
 **R3**
   Each module should define the following variables in its global scope::
-  
+
     __RCSID__ = "$Id$"
 
-  this is the SVN macro substituted by the module revision number. 
-  
+  this is the SVN macro substituted by the module revision number.
+
```

Comment:
Remove this please.

Reply:
There is one more mention of the SVN for the RCSID bellow. SHould I change that to 'git macro' too ?

Reply:
Sure, why not?